From: Pacho Ramos <pacho@gentoo.org>
Date: Wed, 06 Jan 2010 14:17:24 +0000
Subject: Revert upstream commit 7dbe33b5bfd99ff4ef116f8ccf4777edd8ade788
(Chdir to / in the factory, so we don't hog any mountpoints.)
Discussed in:
https://bugzilla.gnome.org/show_bug.cgi?id=583616
http://bugs.gentoo.org/show_bug.cgi?id=292867
---
--- lib/ephy-file-chooser.c~	2009-05-28 21:26:21.000000000 +0200
+++ lib/ephy-file-chooser.c	2010-01-06 15:13:23.000000000 +0100
@@ -375,7 +375,6 @@
 	 * run.
 	 */
 
-        gtk_file_chooser_set_current_folder (GTK_FILE_CHOOSER (dialog), g_get_home_dir ());
 
 	if (persist_key != NULL)
 	{
--- lib/ephy-file-helpers.c~	2009-05-28 21:26:21.000000000 +0200
+++ lib/ephy-file-helpers.c	2010-01-06 15:14:09.000000000 +0100
@@ -33,7 +33,6 @@
 #include "ephy-string.h"
 
 #include <glib.h>
-#include <glib/gstdio.h>
 #include <glib/gi18n.h>
 #include <gio/gio.h>
 #include <gio/gdesktopappinfo.h>
@@ -264,10 +263,6 @@
 {
 	const char *uuid;
 
-	/* Make sure the server process doesn't hog any mountpoints! */
-	if (chdir ("/") < 0)
-		g_warning ("Failed to chdir to /: %s", g_strerror (errno));
-
 	/* See if we've been calling ourself, and abort if we have */
 	uuid = g_getenv (EPHY_UUID_ENVVAR);
 	if (uuid && strcmp (uuid, EPHY_UUID) == 0)
